src/modules/Course/containers/CourseOverview/SectionPresentation.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { ComponentProps } from 'react';
+
+const SectionPresentation: React.FC<ComponentProps<'div'>> = ({
+  children,
+  ...props
+}) => {
+  return (
+    <div {...props}>
+      <h3 className="text-xl text-gray-900 dark:text-white">
+        Apresentação do Curso
```

Comment:
Aren't we going to use the i18n translations?

Reply:
I need to implement the namespace loading per module before use the i18n here or anywhere else.